src/finalizer/index.ts

```diff
@@ -284,9 +291,11 @@ export async function finalize(
       const { miscReason } = crossChainTransfer;
       const originationNetwork = getNetworkName(originationChainId);
       const destinationNetwork = getNetworkName(destinationChainId);
+      const infoLogMessage =
+        amount && symbol ? `to support a ${originationNetwork} withdrawal of ${amount} ${symbol} 🔜` : "";
       logger.info({
         at: "Finalizer",
-        message: `Submitted ${miscReason} on ${destinationNetwork} to support a ${originationNetwork} withdrawal of ${amount} ${symbol} 🔜`,
+        message: `Submitted ${miscReason} on ${destinationNetwork} ` + infoLogMessage,
```

Comment:
```suggestion
        message: `Submitted ${miscReason} on ${destinationNetwork}`,
        infoLogMessage,
```
This way we can record in GCP but keep `message` relatively small.